### PR TITLE
Fork.feat.header&home

### PR DESCRIPTION
**Pull request by miiwu**
- Header
- Home, Repositories
- Home, Recent activity
- Home, All activity
- Home, Explore repositories

### CODE REVIEW

**Comment by miiwu** [comment]
@CharlieEtienne 
Hi, Is this ok?
This is for one PR.

File: material-github.user.css [hunk 1]
```diff
@@ -1111,4 +1111,69 @@
...
+	}
+	.link-gray:hover {
+		color: var(--selected) !important;
+	}
+
+	/* Home, All activity */
+	.dashboard .js-all-activity-header + div {
+		background-color: var(--bg2) !important;
+		border-color: var(--bg2) !important;
+	}
```
Comment (CharlieEtienne) [inline material-github.user.css:1161]: please delete that, it causes a padding issue on home page
2020-10-29_17h36_21

File: material-github.user.css [hunk 2]
```diff
@@ -1111,4 +1111,69 @@
...
+	.js-details-target.js-show-more-recent-items {
+		background-color: var(--bg2) !important;
+		color: rgba(255, 255, 255, calc(0.6 * /*[[text_brightness]]*/)) !important;
+	}
+	.Box-row--hover-blue:hover {
+		background-color: var(--bg3) !important;
+	}
+	.link-gray:hover {
+		color: var(--selected) !important;
+	}
```
Comment (CharlieEtienne) [inline material-github.user.css:1155]: Could you explain what you are trying to achieve here? I don't see any difference when I test your code

File: material-github.user.css [hunk 3]
```diff
@@ -1111,4 +1111,69 @@
...
+	.js-news-feed-event-group.border-bottom {
+		border-bottom: 1px solid var(--bg4) !important;
+	}
+	.js-news-feed-event-group.border-bottom .d-flex.d-flex.flex-items-baseline.py-3 {
+		border-bottom: none !important;
+	}
+	.d-flex.flex-items-baseline.py-3 {
+		border-bottom: 1px solid var(--bg4) !important;
+	}
```
Comment (CharlieEtienne) [inline material-github.user.css:1173]: Honestly, I prefer the current borders, more discrete.

File: material-github.user.css [hunk 4]
```diff
@@ -1111,4 +1111,69 @@
...
+
+	/* Home, Explore repositories */
+	.py-2.my-2.border-bottom {
+		border-color: var(--bg2) !important;
+	}
```
Comment (CharlieEtienne) [inline material-github.user.css:1178]: You can't do that. You are targeting all elements with padding and margin = 2. It's unpredictable. 
Please find another way.

**Review by CharlieEtienne — CHANGES_REQUESTED** [review]

**Comment by miiwu** [comment]
@CharlieEtienne 
Sorry, it would be delayed for several monthes, until i'm leisure.
Or you can close it.

- - -

I'm hobbyist.
I am applying myself to my own major, workingly and intently. 
There may be too much conflicts due to my knowledge shortage, or, github changes itself, just maybe.

**Pull request closed without merging**